.eslintignore

```diff
@@ -0,0 +1,4 @@
+pjax.js
+node_modules/**
```

Comment:
No need for this. In 
https://eslint.org/docs/user-guide/configuring#eslintignore
you can read

> # /node_modules/* and /bower_components/* in the project root are ignored by default

I think it's the same for prettier.

Something I do sometimes is to symlink this *ignore to .gitignore (eg: that's what I do here https://github.com/MoOx/eslint-config-i-am-meticulous)